### ISSUE TITLE
Windows 10 / IE Edge download failed

### ISSUE THREAD

**Issue by PixelHeroMedia** [xdissent/ievms#312]
Happens when installing Edge under any circumstances. IE 10 and 11 work fine.
This is on  a Mac
The download path takes me to a page saying

```
<Error>
<Code>BlobNotFound</Code>
<Message>
The specified blob does not exist. RequestId:60c9de73-0001-00b5-767f-201ce2000000 Time:2016-10-07T09:46:55.1336656Z
</Message>
</Error>
```

This is the error when trying to install

```
Check failed - redownloading OVA ZIP
Downloading OVA ZIP from https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to MSEdge_Win10.zip (attempt 3 of 3)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   215  100   215    0     0    777      0 --:--:-- --:--:-- --:--:--   778
MD5 check failed for MSEdge_Win10.zip (wanted c1011b491d49539975fb4c3eeff16dae, got a5a5da8aa2fb4523a21e1897e070c79c)
Failed to download https://az792536.vo.msecnd.net/vms/VMBuild_20150801/VirtualBox/MSEdge/Mac/Microsoft%20Edge.Win10.For.Mac.VirtualBox.zip to /Users/richard.edwards/.ievms/MSEdge_Win10.zip (attempt 3 of 3)
```

Thanks

**Comment by willpaige** [comment]
I'm getting the same error. Anyone found a solution??

**Comment by thbar** [comment]
Same error - I ultimately had to download the VirtualBox VM directly from https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/.

**Comment by xdissent** [comment]
Should be fixed in 0.3.2. Please reopen if you still have trouble!
